.circleci/config.yml

```diff
@@ -0,0 +1,194 @@
+  save_yarn_cache: &save_yarn_cache
+    save_cache:
+      key: yarn-packages-{{ checksum "client/yarn.lock" }}
+      paths:
+        - ~/.cache/yarn
+  install_tox: &install_tox
+    run: sudo pip install tox
+  set_workdir: &set_workdir
+    working_directory: ~/repo
+  get_code_and_branch: &get_code_and_branch
+    requires:
+      - get_code
+    filters:
+      branches:
+        only:
+          circle-ci
+jobs:
+  get_code:
+    docker:
+      - image: circleci/python:2.7.15
+    <<: *set_workdir
+    steps:
+      # Replace standard code checkout with shallow clone to speed things up.
+      - run:
+          name: Checkout code
+          command: |-
+              # add github.com to known hosts
+              mkdir -p ~/.ssh
+              echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
+              ' >> ~/.ssh/known_hosts
+
+              # add the user ssh key and set correct perms
+              umask 077
+              echo "$CHECKOUT_KEY" > ~/.ssh/id_rsa
+              chmod 0600 ~/.ssh/id_rsa
+
+              # use git+ssh instead of https
+              git config --global url."ssh://git@github.com".insteadOf "https://github.com" || true
+              git config --global gc.auto 0 || true
+
+              # get shallow clone of single repo branch for speed purposes
+              git clone --depth=1 --single-branch -b $CIRCLE_BRANCH git@github.com:$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME ~/repo
+      - save_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/repo
+  py27_lint:
+    docker:
+      - image: circleci/python:2.7.15
+    <<: *set_workdir
+    steps:
+      - *restore_repo_cache
+      - *install_tox
+      - run: tox -e py27-lint
+  py27_unit:
+    docker:
+      - image: circleci/python:2.7.15
+    <<: *set_workdir
+    steps:
+      - *restore_repo_cache
+      - *install_tox
+      - run: tox -e py27-unit
+  py27_docstring:
+    docker:
+      - image: circleci/python:2.7.15
+    <<: *set_workdir
+    steps:
+      - *restore_repo_cache
+      - *install_tox
+  py27_1startup:
+    docker:
+      - image: circleci/python:2.7.15
+    <<: *set_workdir
+    steps:
+      - *restore_repo_cache
+      - run: sh scripts/common_startup.sh
+      - run: wget -q https://github.com/jmchilton/galaxy-downloads/raw/master/db_gx_rev_0127.sqlite
+      - run: mv db_gx_rev_0127.sqlite database/universe.sqlite
+      - run: sh manage_db.sh -c ./config/galaxy.yml.sample upgrade
+      - *install_tox
+      - run: tox -e py27-first_startup
+      - run: tox -e py27-lint_docstring_include_list
```

Comment:
This line should be moved to the `py27_docstring:` section.

Reply:
haha, yeah